src/components/avatar/AvatarInitials.js

```diff
@@ -38,9 +39,10 @@ type Props = {
 
 const AvatarInitials = ({ className = '', id, name }: Props) => {
     const avatarColorSelector = parseInt(id, 10) || 0;
-    const backgroundColor = AVATAR_COLORS[avatarColorSelector % AVATAR_COLORS.length];
+    const backgroundColorIndex = avatarColorSelector % AVATAR_COLORS.length;
```

Comment:
Can we do this math in the SCSS file to avoid needing the list length entirely? We could set the id itself as a `data-` attribute, instead.

Reply:
I tried going down this route but I would then need to add some untestable(?) sass logic to parse the id since attributes are strings.